src/http/web_socket/protocol.cr

```diff
@@ -1,5 +1,5 @@
 require "socket"
-require "http"
+require "http/client"
```

Comment:
What about `http/headers`?

Reply:
Right

Reply:
`require "./client"` and `require "./headers"` would be better BTW

Reply:
in fact, not strictly necessary because it is bring by `http/client` - but I will add it.